src/Game.cpp

```diff
@@ -453,10 +458,21 @@ void Game::LoadLandscape(const std::string& name)
 	_landIsland->LoadFromFile(*file);
 }
 
-const std::string& Game::GetGamePath()
+void Game::SetGamePath(const std::string &gamePath)
 {
-	static std::string sGamePath;
+	if (gamePath.empty())
+	{
+		return;
+	}
+	if (!fs::exists(gamePath))
+	{
+		return;
```

Comment:
Since we were passed some non-null value here and perform a check, it would be nice to report this mishap to the user.